teamcity/datadog_checks/teamcity/teamcity.py

```diff
@@ -110,6 +103,13 @@ def check(self, instance):
         if instance_name is None:
             raise Exception("Each instance must have a unique name")
 
+        # Use HTTP or HTTPS
+        protocol = 'http'
+        use_https = _is_affirmative(instance.get('use_https', False))
+        if use_https:
+            protocol = 'https'
```

Comment:
these could be `protocol = 'https' if _is_affirmative(instance.get('use_https', False)) else 'http'`